### ISSUE TITLE
Change the rule name on the details tab to an H3 + add hidden H4's for the rules

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/accessibility-checker#76]
```
<div class="edac-details-rule-title">
<h3>
<span class="edac-details-rule-count edac-details-rule-count-error active">9</span>Insufficient Color Contrast
</h3>
[If there are ignores, this will go here, outside the headings]
</div>
```

Then, inside the div for the code

```
<div class="edac-details-rule-records-record-cell edac-details-rule-records-record-object"><code>&lt;a href="#"&gt;odio dignissim qui blandit&lt;/a&gt;</code></div>
```

Add a screen reader H4 that says Rule Name Issue 1, Rule Name Issue 2, etc. - this will make it easier for screen reader users to get through the rules.

**Comment by SteveJonesDev** [comment]
Updated since version 1.3.10
